src/rpc/blockchain.cpp

```diff
@@ -1243,8 +1242,9 @@ static void BIP9SoftForkDescPushBack(UniValue& softforks, const std::string &nam
     {
         bip9.pushKV("bit", consensusParams.vDeployments[id].bit);
     }
-    bip9.pushKV("start_time", consensusParams.vDeployments[id].nStartTime);
-    bip9.pushKV("timeout", consensusParams.vDeployments[id].nTimeout);
+    bip9.pushKV("startheight", consensusParams.vDeployments[id].startheight);
+    bip9.pushKV("timeoutheight", consensusParams.vDeployments[id].timeoutheight);
+    bip9.pushKV("minimum_activation_height", consensusParams.vDeployments[id].m_min_activation_height);
```

Comment:
Probably should either hide this when 0, or use starttime+2*period

Reply:
I think it's fine to always display.